endpointmanager/pkg/nppesquerier/populatenpicontacts.go

```diff
@@ -100,6 +104,53 @@ func isValidURL(url string) bool {
 	return urlmatched && !emailmatched
 }
 
+func removeNestedDoubleQuotesFromCSV(filename string) (error, string) {
+	file, err := os.Open(filename)
+
+	if err != nil {
+		return err, ""
+	}
+
+	scanner := bufio.NewScanner(file)
+	scanner.Split(bufio.ScanLines)
+	var processedlines []string
+
+	for scanner.Scan() {
+		line := scanner.Text()
+		// TODO: This regex is only checking every other field starting with the second feld,
+		// it is capturing all of the free-text fields that include bad-data as of the March/2020 NPI Data
```

Comment:
that's convenient... but makes me wary.
it looks like this may do the reverse so you could combine both of these to cover the whole file: `(")(.*?)","`
![image](https://user-images.githubusercontent.com/11574199/80702013-5ef0bf80-8aae-11ea-8447-e9f895577f01.png)

Reply:
I combined 2 regexes, but the regex you suggested wouldn't work because it captures the outer quotations as part of the value group.